### PR TITLE
商品詳細ページに同じカテゴリの商品を表示する機能を追加しました

### PR DESCRIPTION
**Pull request by SekiMoemi**
以下のように修正しました。ご確認お願いします。

・メソッド名の先頭が大文字になっている箇所を修正しました
・商品詳細を取得する際に、同ジャンルの商品リストも取得するように修正しました
・ItemのgenreをInteger型に変更しました
・Itemリポジトリでgenreを取得するように修正しました
・商品詳細ページに3件ずつ同じカテゴリの商品が表示されるようにHTMLを修正しました

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 
+import jp.co.example.ecommerce_a.constant.Genre;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'jp.co.example.ecommerce_a.constant.Genre'

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:11]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'jp.co.example.ecommerce_a.constant.Genre' import. Should be before 'org.springframework.web.bind.annotation.RequestMapping'.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java [hunk 2]
```diff
@@ -37,9 +38,14 @@ public class ShowItemDetailController {
 	@GetMapping("")
 	public String ShowItemDetail(String itemId, Model model) {
 		final int NUM_OF_SHOW_ITEM = 3;
-		model.addAttribute("item", showItemDetailService.showItemDetail(Integer.parseInt(itemId)));
-		List<Item> itemList = showItemListService.searchByName("", "0").stream().limit(NUM_OF_SHOW_ITEM).toList();
-		model.addAttribute("itemList", itemList);
+		Item item = showItemDetailService.showItemDetail(Integer.parseInt(itemId));
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java [hunk 3]
```diff
@@ -37,9 +38,14 @@ public class ShowItemDetailController {
...
+		model.addAttribute("item", item);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:42]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:42]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java [hunk 4]
```diff
@@ -37,9 +38,14 @@ public class ShowItemDetailController {
...
+		List<Item> popularItemList = showItemListService.searchByName("", "0").stream().limit(NUM_OF_SHOW_ITEM)
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 119).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java [hunk 5]
```diff
@@ -37,9 +38,14 @@ public class ShowItemDetailController {
...
+				.toList();
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java [hunk 6]
```diff
@@ -37,9 +38,14 @@ public class ShowItemDetailController {
...
+		model.addAttribute("popularItemList", popularItemList);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:45]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java [hunk 7]
```diff
@@ -37,9 +38,14 @@ public class ShowItemDetailController {
...
+		List<Item> genreItemList = showItemListService.narrowingDown(null, item.getGenre(), null);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 106).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:46]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java [hunk 8]
```diff
@@ -37,9 +38,14 @@ public class ShowItemDetailController {
...
+		model.addAttribute("genreItemList", genreItemList);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java [hunk 9]
```diff
@@ -37,9 +38,14 @@ public class ShowItemDetailController {
...
+		model.addAttribute("genre", Genre.of(item.getGenre()).getValue());
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:48]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowItemDetailController.java:48]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 10]
```diff
@@ -35,7 +35,7 @@ public class ItemRepository {
 	 * @return 全おもちゃのリスト
 	 */
 	public List<Item> findAll() {
-		String sql = "Select id, name, description, price_m, price_l, image_path, deleted from items;";
+		String sql = "Select id, name, description, price_m, price_l, image_path, deleted, genre from items;";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:38]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 118).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:38]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:38]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 11]
```diff
@@ -47,7 +47,7 @@ public List<Item> findAll() {
 	 * @return 検索された全おもちゃのリスト
 	 */
 	public List<Item> findByName(String name) {
-		String sql = "Select id, name, description, price_m, price_l, image_path, deleted from items where name ILIKE :name ORDER BY id;";
+		String sql = "Select id, name, description, price_m, price_l, image_path, deleted, genre from items where name ILIKE :name ORDER BY id;";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:50]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 153).

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:50]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:50]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 12]
```diff
@@ -57,46 +57,47 @@ public List<Item> findByName(String name) {
 	/**
 	 * 対象年齢、ジャンル、価格帯でおもちゃを絞り込む.
 	 * 
-	 * @param targetAge　対象年齢
-	 * @param genre　ジャンル
-	 * @param priceRange　価格帯
-	 * @return　絞り込みされたおもちゃのリスト
+	 * @param targetAge  対象年齢
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:60]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.RequireEmptyLineBeforeBlockTagGroupCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Javadoc tag '@param' should be preceded with an empty line.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:60]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 13]
```diff
@@ -57,46 +57,47 @@ public List<Item> findByName(String name) {
...
+	 * @param genre      ジャンル
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:61]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 14]
```diff
@@ -57,46 +57,47 @@ public List<Item> findByName(String name) {
...
+	 * @param priceRange 価格帯
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:62]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java [hunk 15]
```diff
@@ -57,46 +57,47 @@ public List<Item> findByName(String name) {
...
+	 * @return 絞り込みされたおもちゃのリスト
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

**Review by SekiMoemi — COMMENTED** [review]
Remaining comments which cannot be posted as a review comment to avoid GitHub Rate Limit

<details>
<summary>checkstyle</summary>

[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|66|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L66) Line is longer than 100 characters (found 123).
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|66 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L66) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|66 col 17|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L66) Distance between variable 'sql' declaration and its first usage is 4, but allowed 3.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value).
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|66 col 17|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L66) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|67 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L67) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|67 col 17|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L67) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|68 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L68) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|68 col 17|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L68) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|69 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L69) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|69 col 17|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L69) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|72 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L72) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|72 col 17|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L72) 'if' has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|76 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L76) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|76 col 17|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L76) 'if' has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|80 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L80) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|80 col 17|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L80) 'if' has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|92 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L92) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|93 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L93) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|100|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L100) Line is longer than 100 characters (found 131).
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|100 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L100) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|100 col 17|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L100) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|112 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L112) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|112 col 9|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L112) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|114|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L114) Line is longer than 100 characters (found 122).
[src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java|114 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/repository/ItemRepository.java#L114) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java|34 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java#L34) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java|34 col 17|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/service/ShowItemListService.java#L34) 'method def' child has incorrect indentation level 16, expected level should be 4.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|30 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L30) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|30 col 9|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L30) 'member def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|84|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L84) Line is longer than 100 characters (found 128).
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|84 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L84) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|155 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L155) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|155 col 9|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L155) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|159 col 1|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L159) Line contains a tab character.
[src/main/java/jp/co/example/ecommerce_a/domain/Item.java|159 col 9|](http://github.com/ecsite-project/ec-202304a/blob/43fe9b4f9e7b973c4d0e7422a8ae2ce25170b9b7/src/main/java/jp/co/example/ecommerce_a/domain/Item.java#L159) 'method def modifier' has incorrect indentation level 8, expected level should be 2.
</details>

**Review by yamada-kensyu — APPROVED** [review]
承認します！

**Pull request merged**